crud/src/main/java/com/crud/controller/employeeController.java

```diff
@@ -0,0 +1,105 @@
+/**
+ * 
+ */
+package com.crud.controller;
+
+import java.sql.Date;
+import java.util.List;
+import java.util.Optional;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.crud.entity.Department;
+import com.crud.entity.Employee;
+import com.crud.service.EmployeeService;
+
+/**
+ * 
+ */
+
+@RestController
+@RequestMapping("/")
+public class employeeController {
+
+	@Autowired
+	EmployeeService service;
+
+	@PostMapping("/saveEmployee")

```

Comment:
already a post mapping, don't need to specify save here. It can simply employee.